### PR TITLE
[Docs] Update Japanese translation of keymap.md.

### PR DESCRIPTION
**Pull request by shelaf**
## Description

Japanese documentation update.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

tag: @mtei

### CODE REVIEW

**Comment by mtei** [comment]
@umi-umi @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/keymap.md [hunk 1]
```diff
@@ -76,10 +76,22 @@ TMK の歴史的経緯から、キーマップに保存されたアクション
...
+        /___________/
+    
+    上記シナリオでは、上位レイヤーにある非透過のキーを使うことができますが、`KC_TRNS` (または同等のキーコード)が定義されている場合は常に下位レベルのキーコード(`KC_A`)が使われます。
+
+**メモ:** 特定のレイヤーの透過性を示す有効な方法:
+* `KC_TRANSPARENT`
+* `KC_TRNS` (別名)
+* `_______` (別名)
+
+これらのキーコードは、処理する非透過のキーコードを検索して、処理を下位レイヤーに移すことができます。
```
Comment (mtei) [inline docs/ja/keymap.md:94]: これまた、原文よくわかってなくて、推測です。

```suggestion
これらのキーコードは、処理する非透過のキーコードを探すために、検索を下位レイヤーに移すことができます。
```

File: docs/ja/keymap.md [hunk 2]
```diff
@@ -76,10 +76,22 @@ TMK の歴史的経緯から、キーマップに保存されたアクション
...
 
-オーバーレイレイヤーに `KC_TRANS`  を配置して、レイアウトの一部だけを変更して下位レイヤーまたは基本レイヤーにフォールバックすることができます。
-`KC_TRANS` (`KC_TRNS` と `_______` はエイリアス) のキーには独自のキーコードがなく、キーコードの有効な下位レイヤーを参照します。
+           ____________
+          /           /  <--- Higher layer
+         /  KC_TRNS  //
+        /___________//   <--- Lower layer (KC_A)
+        /___________/
+    
+    上記シナリオでは、上位レイヤーにある非透過のキーを使うことができますが、`KC_TRNS` (または同等のキーコード)が定義されている場合は常に下位レベルのキーコード(`KC_A`)が使われます。
```
[the remark this answers is not in the document]
Comment (shelaf) [reply docs/ja/keymap.md]: 次の 〜 is defined に合わせて、〜が定義されていると、としてみました。

File: docs/ja/keymap.md (hunk 1 above)
Comment (shelaf) [reply docs/ja/keymap.md]: もう少しわかり易くしてみたのですがどうでしょうか

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by noroadsleft** [comment]
ありがとうございました！

**Pull request merged**